code/defines/procs/announce.dm

```diff
@@ -137,20 +129,10 @@
 	if(message_announcer)
 		formatted_message += "<br><span class='alert'> -[message_announcer]</span>"
 	formatted_message += "<br>"
-	
-	return formatted_message
 
-/datum/announcement/priority/command/Format_Message(message, message_title, message_announcer, from)
-	var/formatted_message
-	formatted_message += "<h1 class='alert'>[title]</h1>"
-	if(message_title)
-		formatted_message += "<br><h2 class='alert'>[message_title]</h2>"
-	formatted_message += "<br><span class='alert'>[message]</span><br>"
-	formatted_message += "<br>"
-	
 	return formatted_message
 
-/datum/announcement/priority/enemy/Format_Message(message, message_title, message_announcer, from)
```

Comment:
All admin announcements now use `/datum/announcement/priority/command`, with changing titles, sounds and all that, so there's not much point in keeping the enemy announcement thingy here.